lib/foods.js

```diff
@@ -1,65 +1,75 @@
 $('document').ready(function() {
+  for (var i = 0; i < localStorage.length; i++){
+    $("tbody").append(localStorage.getItem(localStorage.key(i)));
```

Comment:
This is nice! I like that it will only run when the page is loaded. Will it add the `<td>`s as well?